Screens/ProgramsInfo.js

```diff
@@ -0,0 +1,183 @@
+import * as React from "react";
```

Comment:
What is the difference between ProgramsInfo.js and Course.js? it seems like they do the same thing. You can decide which to keep.

Reply:
No difference. ProgramsInfo.js will be deleted after merging.